File: data/CodeLlama/readable_format_HPEC/fortran/file094.f90
program DRB115_forsimd_orig_yes
    use omp_lib
    implicit none

    integer i, len
    integer, dimension(:), allocatable :: a,b

    len = 100
    allocate (a(len))
    allocate (b(len))

    do i = 1, len
        a(i) = i
        b(i) = i+1
    end do

    !$omp simd
    do i = 1, len-1
        a(i+1)=a(i)+b(i)
    end do

    print*,'a(50) =',a(50)

    deallocate(a,b)
end program

File: data/gpt-4o/readable_format_HPEC/cpp/file094.cpp
#include <stdio.h>
int a[100];
int b[100];
int main()
{
 int i;
 for (i=0;i<100;i++)
 {
 a[i] = i+1;
 b[i] = i+2;
 }
#pragma omp simd
 for (i=0;i<99;i++)
 a[i+1] = a[i] + b[i];

 printf ("a[50]=%d\n", a[49]);
 return 0;
}
